backend/apps/recommendations/sync_oss_issues.py

```
@@ -0,0 +1,55 @@
+            headers['Authorization'] = f'token {GITHUB_TOKEN}'
+        
+        # Search for good first issues
+        url = 'https://api.github.com/search/issues'
+        params = {
+            'q': 'label:"good first issue" is:open',
+            'sort': 'updated',
+            'per_page': 100,
+        }
+        
+        try:
+            response = requests.get(url, headers=headers, params=params)
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)